source/hwPortUtils.py

```diff
@@ -231,7 +231,9 @@ def __str__(self):
 			):
 				# Ignore ERROR_INSUFFICIENT_BUFFER
 				if ctypes.GetLastError() != ERROR_INSUFFICIENT_BUFFER:

```

Comment:
Unrelated to your change. While it might be unlikely I think it would be a good idea to log an insufficient buffer error.